tests/mixins/test_library.py

```diff
@@ -109,3 +111,9 @@ def test_rate_playlist(self, yt_auth):
     def test_subscribe_artists(self, yt_auth):
         yt_auth.subscribe_artists(["UCUDVBtnOQi4c7E8jebpjc9Q"])
         yt_auth.unsubscribe_artists(["UCUDVBtnOQi4c7E8jebpjc9Q"])
+
+    def test_get_account_info(self, config, yt_oauth):
```

Comment:
Please test with a non-authenticated account

If it doesn't work, make sure to raise an Exception in the function and expect it here with `pytest.raises`